### PR TITLE
Linux 9253: pylint fixes

### PR DESCRIPTION
**Pull request by ejannett**
unit test passed 
```
[root@00-manu-dev oci-utils]# python3 setup.py oci_tests --tests-base tests/data/
INFO:oci-utils.TreeConfigParser:all properties files read : ['tests/data/properties.cfg']
ok
test_get_network_config (tests.test_vnicutils.TestVnicUtils) ... all properties files read : ['tests/data/properties.cfg']
INFO:oci-utils.TreeConfigParser:all properties files read : ['tests/data/properties.cfg']
{'CONFSTATE': '-', 'IS_PRIMARY': 'True', 'MAC': '02:00:17:00:35:E2', 'ADDR': '100.110.4.83', 'SPREFIX': '100.110.4.0', 'SBITS': '24', 'VIRTRT': '100.110.4.1', 'VLTAG': '2754', 'VNIC': 'ocid1.vnic.oc1.uk-london-1.abwgiljr2zdrk6frn4evor7xdejwp5f2cqrxn5aysilnmsyoun3ihi7hwzga', 'IFACE': 'ens3', 'LINKTYPE': 'ether', 'IND': '2', 'STATE': 'UP'}
{'CONFSTATE': '-', 'MAC': '52:54:00:EC:90:0E', 'IFACE': 'virbr0-nic', 'LINKTYPE': 'tun', 'IND': '4', 'STATE': 'DOWN'}
ok

----------------------------------------------------------------------
Ran 71 tests in 110.441s

OK (skipped=8)
[root@00-manu-dev oci-utils]# 
```

### CODE REVIEW

File: lib/oci_utils/impl/oci-metadata-main.py [hunk 1]
```diff
@@ -174,9 +171,9 @@ def pretty_print_section(metadata, indent):
         No return value.
     """
     # first display the keys that are in the oci_metadata_display_order list
-    if type(metadata) is not dict:
+    if isinstance(metadata, dict):
```
Comment (guidotijskens) [inline lib/oci_utils/impl/oci-metadata-main.py:174]: suppose this should be 
if not isinstance( ....

File: lib/oci_utils/impl/oci-network-config-main.py [hunk 2]
```diff
@@ -536,14 +530,11 @@ def main():
...
-        (ret, out) = vnic_utils.auto_config(quiet=args.quiet, show=args.show,
-                                            sec_ip=args.sec_ip)
+        (ret, out) = vnic_utils.auto_config(args.sec_ip)
     elif args.deconfigure:
-        (ret, out) = vnic_utils.auto_deconfig(quiet=args.quiet, show=args.show,
-                                              sec_ip=args.sec_ip)
+        (ret, out) = vnic_utils.auto_deconfig(args.sec_ip)
 
     if ret:
         __logger.error("Failed to execute the VNIC configuration script.")
```
Comment (guidotijskens) [inline lib/oci_utils/impl/oci-network-config-main.py:540]: maybe the opportunity to change __logger to _logger in this module

Comment (ejannett) [reply lib/oci_utils/impl/oci-network-config-main.py:540]: fixed

File: lib/oci_utils/metadata.py [hunk 3]
```diff
@@ -274,39 +270,30 @@ def _name_convert_underscore(self, meta):
...
-        if 'vnics' in metadata:
-            for i in range(len(metadata['vnics'])):
-                v = metadata['vnics'][i].pop('id')
-                metadata['vnics'][i]['vnicId'] = v
-        return metadata
+        if 'vnics' in self._metadata:
+            for i in range(len(self._metadata['vnics'])):
+                v = self._metadata['vnics'][i].pop('id')
+                self._metadata['vnics'][i]['vnicId'] = v
+
```
Comment (guidotijskens) [inline lib/oci_utils/metadata.py:296]: did this ever work?

Comment (ejannett) [reply lib/oci_utils/metadata.py:296]: I hope so. you should ask this to the owner of these lines.

File: lib/oci_utils/metadata.py [hunk 4]
```diff
@@ -627,8 +572,8 @@ def refresh(self, get_public_ip=False):
         if metadata:
             self._metadata = OCIMetadata(metadata)
 
-        #_logger.debug('metadata after refresh [%s]' % str(self._metadata))
-        return result
+        return self
+
```
Comment (guidotijskens) [inline lib/oci_utils/metadata.py:576]: no danger for read-consistency problems?

Comment (ejannett) [reply lib/oci_utils/metadata.py:576]: We have in many solace use of locking and in many other nothing....  all this for mono-threaded programs. 
I've choose to remove locking and re-think about it if we need .

File: lib/oci_utils/impl/ocid-main.py [hunk 5]
```diff
@@ -153,6 +149,10 @@ def getName(self):
         return self.thread_name
 
     def request_stop(self):
+        """
+        trigger a stop phase of the daemon
+        set the event triggering the stop
```
Comment (guidotijskens) [inline lib/oci_utils/impl/ocid-main.py:154]: ... then stop

**Review by guidotijskens — APPROVED** [review]
i like this

**Pull request merged**